test/features/service_discovery.feature

```diff
@@ -20,8 +20,11 @@ Feature: Discovery with onboarding
       | label  | payments/date  | process  |
 
     And Deploy quarkus example service "onboarding-example/onboarding" with configuration:
-      | config | native     | <native> |
-      | label  | onboarding | process  |
+      | config        | native     | <native> |
+      | label         | onboarding | process  |
+      | build-request | cpu        | 4        |
+      | build-request | memory     | 10Gi     |
+      | build-limit   | cpu        | 8        |
```

Comment:
Is this build that demanding? Just asking out of curiosity.

Reply:
I think there are a couple of issues here:
- Why the onboarding project requires so amount of memory to build? Unlikely other quarkus apps which only require 4 Gi (the problem could be because the cloud-events dependency). However, I tried many tweaks and none made the native build require less memory :(
- The another issue is that we're triggering a few times the same build because of the bug [I raised last week ](https://issues.redhat.com/browse/KOGITO-1771)which make the resources more busy

Reply:
@Sgitario I think it would have sense to parametrize the resources too (not just native flag) as non native build doesn't have so large requirements - can be set much lower.
That can save resources.

Reply:
it makes totally sense this. Will do.

Reply:
Done